spec/javascripts/components/step-by-step-nav-spec.js

```diff
@@ -4,9 +4,9 @@
 describe('A stepnav module', function () {
   'use strict'
 
-  var $element
+  var $element, element
```

Comment:
I'm using `element` here as I'm not using jQuery like the other tests, and variables starting with `$` indicate they're a jQuery variable, so I didn't use it.